protocol/test/forge/v2/interestRate/CompoundedInterest.t.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import { TempleTest } from '../../TempleTest.sol';
+import { CompoundedInterest } from 'contracts/v2/interestRate/CompoundedInterest.sol';
+import { CommonEventsAndErrors } from 'contracts/common/CommonEventsAndErrors.sol';
+
+import { stdError } from "forge-std/Test.sol";
+import { PRBMath_MulDiv18_Overflow } from "@prb/math/src/Common.sol";
+
+contract CompoundedInterestTestBase is TempleTest {
+  using CompoundedInterest for uint256;
+  uint256 public constant ONE_PCT_INTEREST = 0.01e18;
```

Comment:
You could define this as a `uint96` to save having to cast it below (same as for the FIVE_PCT and TEN_PCT)